transport-native-epoll/src/main/c/io_netty_channel_unix_FileDescriptor.c

```diff
@@ -24,3 +25,16 @@ JNIEXPORT int JNICALL Java_io_netty_channel_unix_FileDescriptor_close(JNIEnv* en
    }
    return 0;
 }
+
+JNIEXPORT int JNICALL Java_io_netty_channel_unix_FileDescriptor_open(JNIEnv* env, jclass clazz, jstring path) {
+
+    const char* f_path = (*env)->GetStringUTFChars(env, path, 0);
+
+    int res = open(f_path, O_WRONLY | O_CREAT | O_TRUNC, 0644);
```

Comment:
Yes it does , see the umask manpage:

http://linux.die.net/man/2/umask

I wonder if because of this 666 would be better though... @trust WDYT?



> Am 30.04.2015 um 02:57 schrieb Trustin Lee <notifications@github.com>:
> 
> In transport-native-epoll/src/main/c/io_netty_channel_unix_FileDescriptor.c:
> 
> > @@ -24,3 +25,16 @@ JNIEXPORT int JNICALL Java_io_netty_channel_unix_FileDescriptor_close(JNIEnv* en
> >     }
> >     return 0;
> >  }
> > +
> > +JNIEXPORT int JNICALL Java_io_netty_channel_unix_FileDescriptor_open(JNIEnv* env, jclass clazz, jstring path) {
> > +
> > +    const char* f_path = (*env)->GetStringUTFChars(env, path, 0);
> > +
> > +    int res = open(f_path, O_WRONLY | O_CREAT | O_TRUNC, 0644);
> I'm not sure open respects umask. Could you confirm?
> 
> —
> Reply to this email directly or view it on GitHub.
>